### PR TITLE
Added Dockerfile for development. Updated README and Makefile

### PR DESCRIPTION
**Pull request by tyndyll**
Dockerfile added to simplify development process. Instructions for how to run are in the README and a makefile command has been added (`make docker`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,33 @@ environment it is suggested that you use [RVM][rvm].
...
+If `make` is not installed the following two commands should be executed
+
+```
+docker build -t farsetlabs-site .
+docker run -p 4000:4000 -v `pwd`:/app farsetlabs-site
+```
+
+This will start a container, build the site and then serve it on port 4000. 
+Editing files on your machine will cause the container to rebuild the site and
+make the changes available in the browser (wait for the confirmation message of
```
Comment (awflwafl) [inline README.md]: Looks like the closing ")" is missing.

File: about/index.html [hunk 2]
```diff
@@ -71,7 +70,6 @@ <h5 class="subheader">{{officer.title}} ({{officer.year}} )</h5>
             {% endif %}
             <ul class="small button-group round right">
               {% if officer.email %}<li><a class="small button" href="mailto:{{officer.email}}">{{officer.email}} <i class="general foundicon-mail"></i></a></li>{% endif %}
-              {% if officer.discourse %}<li><a class="small button" href="{{site.social.forum}}users/{{officer.discourse}}">Get in touch <i class="social foundicon-torso" style="height:51px"></i></a></li>{% endif %}
```
Comment (awflwafl) [inline about/index.html]: Are these links to discourse intended to be removed as part of this pull request? I'm happy to approve the added Docker support, but can't comment on removing discourse links.

Comment (tyndyll) [reply about/index.html]: Pushed in error - have updated the branch

**Comment by tyndyll** [comment]
Updated the comment, resolved the additional file

**Pull request merged**